websites/W/Wolvesville/presence.ts

```diff
@@ -322,23 +314,24 @@ presence.on("UpdateData", async () => {
                 .querySelector(
                   "div.css-1dbjc4n.r-150rngu.r-eqz5dr.r-16y2uox.r-1wbh5a2.r-5oul0u.r-11yh6sk.r-1rnoaur.r-1sncvnh.r-13qz1uu"
                 )
-                ?.innerHTML.match(/text-decoration-line: line-through/g) || []
+                ?.textContent.match(/text-decoration-line: line-through/g) || []
```

Comment:
why did the `innerHTML`s changed to `textContent`s? 
in this case it breaks the presence.

Reply:
textContent is better to use over innerHTML, maybe in this case it was not appropriate